client/src/components/ElementEditor/InlineEditForm.scss

```diff
@@ -25,4 +25,25 @@
       max-width: 900px;
     }
   }
+
+  // Render the form off the screen so that it's not visible
+  // Note that using display:none will mean the form is not rendered at all and cannot be submitted
+  &--rendered-not-visible {
```

Comment:
There's an `aria-hidden` (or whatever the actual thing is) to avoid this causing problems for screen-reader users.
It needs to be rendered so the form can submit with the correct values. Otherwise, the formschema form submission has no values to submit.

It's an unfortunate quirk of using formschema in an element that can be collapsed.

Reply:
Have put a note about this in Content.js which is where this css class is added